src/main/java/com/example/demo/entities/CarEntity.java

```diff
@@ -0,0 +1,50 @@
+package com.example.demo.entities;
+
+import com.example.demo.enums.Category;
+import com.example.demo.enums.Transmission;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Getter;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@javax.persistence.Entity
+@Table(name = "car")
+public class CarEntity {
+    @Id
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    @Getter public long id;
+    @Getter private String name;
+    @Getter private String registration;
+    @Getter private String manufacturer;
+    @Getter private String model;
+    @Enumerated(EnumType.STRING) @Getter private Transmission transmission;
+    @Enumerated(EnumType.STRING) @Getter private Category category;
+
+    @JsonIgnoreProperties("cars")
+    @ManyToOne(optional = false)
+    @JoinColumn(name = "location_id")
+    @OnDelete(action = OnDeleteAction.CASCADE)
+    @Getter private LocationEntity location;
+
+    //@JsonIgnoreProperties("bookings")
+    @OneToMany(cascade = CascadeType.REMOVE, mappedBy = "car", fetch = FetchType.LAZY)
+    @Getter private Set<BookingEntity> bookings = new HashSet<>();
+
+    public CarEntity(){}
+
+    public CarEntity(String name, String registration, String manufacturer,
```

Comment:
You can get Lombok to do this for you too, along with the empty annotations with the following on your class

    @NoArgsConstructor
    @RequiredArgsConstructor

Reply:
You can also tell it to do getters for every field for you with `@Getter` on the class level